src/pipeline.js

```diff
@@ -146,15 +147,15 @@ class Pipeline {
      * @param {String} name - name of the extension point (typically the function name).
      * @param {pipelineFunction} f - a new pipeline step that will be injected relative to `name`.
      */
-    this.attach.before = (name, f) => this.attach.generic(name, f, 0);
+    this.attach.before = (name, f) => this.attach.generic(name, f, true);
```

Comment:
if `attachGeneric(before, name, f)` is a function, you could write:

```
this.attach.before = attachGeneric.bind(this, true);
```

Reply:
I used a similar approach in 461d3f6